worldwind/src/commonMain/kotlin/earth/worldwind/shape/milstd2525/AbstractMilStd2525TacticalGraphic.kt

```diff
@@ -73,6 +75,18 @@ abstract class AbstractMilStd2525TacticalGraphic(
         // Check if tactical graphics visible
         val terrainSector = rc.terrain.sector
         if (!terrainSector.isEmpty && terrainSector.intersects(sector) && getExtent(rc).intersectsFrustum(rc.frustum)) {
+            if(previousLod != lod) {
+                val previousLodShapes = lodBuffer[previousLod]
+                if(previousLodShapes != null) {
+                    val layer = rc.currentLayer as RenderableLayer
+                    for (renderable in previousLodShapes) {
+                        val renderer = layer.batchRenderers[renderable::class]?: continue
+                        renderer.removeRenderable(renderable)
+                    }
+                }
+            }
+            previousLod = lod
```

Comment:
Can we remove this code and ralay on automatic batch cleanup on each frame?

Reply:
done

Reply:
done